test/check-application

```diff
@@ -1174,6 +1177,20 @@ class TestFiles(testlib.MachineCase):
         b.wait_not_present(".pf-v5-c-modal-box")
         b.wait_visible("[data-item='renamed-foo.txt']")
 
+        # Rename file using keyboard shortcut
+        b.click("[data-item='newfile']")
+        b.key("\uE032")  # F2
```

Comment:
Hmmm any reason "F2" doesn't work?

Reply:
Could work if I add it here in testlib: https://github.com/cockpit-project/cockpit/blob/dc2085e95137ced40e03c94b8b2ebcf090a8acea/test/common/testlib.py#L103

Reply:
Seems worth it, let's do it :) But don't block merging on it.